### PR TITLE
landing page completed

### PR DESCRIPTION
**Pull request by binismail**

### CODE REVIEW

**Review by heyitschun — CHANGES_REQUESTED** [review]
I made a quick change to the `register` button border classes because it jumped on hover. Hope that's okay. Overall looks very good!

I think the footer can use some padding and the menu button on mobile doesn't work for me.

**Comment by priyajitm** [comment]
@heyitschun what is going on with this?

**Comment by heyitschun** [comment]
> @heyitschun what is going on with this?

The menu button doesn't open a menu on mobile. Should that not be a change request?

**Comment by priyajitm** [comment]
> > @heyitschun what is going on with this?
> 
> The menu button doesn't open a menu on mobile. Should that not be a change request?

yes sure it will be. I was asking to get a general idea as I believe I saw another commit added to the PR after your change request. So I thought you might have fixed it.

**Comment by heyitschun** [comment]
> > > @heyitschun what is going on with this?
> > 
> > 
> > The menu button doesn't open a menu on mobile. Should that not be a change request?
> 
> yes sure it will be. I was asking to get a general idea as I believe I saw another commit added to the PR after your change request. So I thought you might have fixed it.

No, they are for different parts of the site.

**Comment by insuusvenerati** [comment]
Checking out this now.

File: zappnode-front/src/landing/components/header/navigation.js [hunk 1]
```diff
@@ -2,41 +2,60 @@ import React, {useState, useEffect} from 'react';
...
+    }
+
+    console.log(props.toogle)
+  }
+
+  
+  useEffect(()=> {
+    console.log('effect fired')
+  }, [handleClick]);
+
```
Comment (insuusvenerati) [inline zappnode-front/src/landing/components/header/navigation.js:16]: ESLint: The 'handleClick' function makes the dependencies of useEffect Hook (at line 25) change on every render.

File: zappnode-front/src/landing/components/header/navigation.js [hunk 2]
```diff
@@ -2,41 +2,60 @@ import React, {useState, useEffect} from 'react';
 import '../css/tailwind.generated.css';
```
Comment (insuusvenerati) [inline zappnode-front/src/landing/components/header/navigation.js]: The css files only needs to be imported once at the root of the app. Like in App.js or Index.js

```suggestion

```

**Comment by binismail** [comment]
i have updated the code too

**Comment by binismail** [comment]
> > @heyitschun what is going on with this?
> 
> The menu button doesn't open a menu on mobile. Should that not be a change request?

am working on that

File: zappnode-front/src/landing/index.js [hunk 3]
```diff
@@ -1,12 +1,17 @@
-import React from 'react';
+import React, {useEffect, useState} from 'react';
```
Comment (insuusvenerati) [inline zappnode-front/src/landing/index.js]: Try to clean up unused imports if you can with `eslint --fix src/`

```suggestion
import React, { useState} from 'react';
```

**Review by insuusvenerati — CHANGES_REQUESTED** [review]
Looks really good so far. Only minor things to change :)

File: zappnode-front/src/landing/components/header/navigation.js (hunk 2 above)
Comment (binismail) [reply zappnode-front/src/landing/components/header/navigation.js]: okay, I will work on that

File: zappnode-front/src/landing/index.js (hunk 3 above)
Comment (binismail) [reply zappnode-front/src/landing/index.js]: okay

File: zappnode-front/src/landing/components/header/navigation.js (hunk 1 above)
Comment (binismail) [reply zappnode-front/src/landing/components/header/navigation.js:16]: yeah, am using it to work on navigation when am done, I will get rid of it

**Comment by binismail** [comment]
please review, will be working on the navigation soon

File: zappnode-front/src/landing/components/navigation/navigate.js [hunk 4]
```diff
@@ -1,30 +1,36 @@
 import React from 'react';
 import '../css/tailwind.generated.css';
 import '../css/landing.css';
```
Comment (insuusvenerati) [inline zappnode-front/src/landing/components/navigation/navigate.js]: Styles should be imported in App.js or Index.js

**Review by insuusvenerati — CHANGES_REQUESTED** [review]
Once the css imports are removed this will be good to go :D

File: zappnode-front/src/landing/components/navigation/navigate.js (hunk 4 above)
Comment (priyajitm) [reply zappnode-front/src/landing/components/navigation/navigate.js]: have you addressed the above-request change? is it good to merge?

**Comment by binismail** [comment]
please review

**Review by heyitschun — APPROVED** [review]

**Review by binismail — COMMENTED** [review]
changes made

**Pull request closed without merging**